plutus-core/plutus-ir/test/PlutusIR/Transform/Inline/nameCapture.golden

```diff
@@ -1,15 +1,15 @@
 (let
   (nonrec)
-  (termbind (strict) (vardecl a (con integer)) (con integer 10))
-  (let
-    (nonrec)
-    (termbind
-      (strict) (vardecl c (con integer)) [ [ (builtin addInteger) a ] a ]
-    )
-    (let
-      (nonrec)
-      (termbind (strict) (vardecl a (con integer)) (con integer 7))
-      [ [ (builtin addInteger) [ [ (builtin addInteger) a ] a ] ] a ]
-    )
+  (termbind
+    (strict)
+    (vardecl c (con integer))
+    [ [ (builtin addInteger) (con integer 10) ] (con integer 10) ]
   )
+  [
+    [
+      (builtin addInteger)
+      [ [ (builtin addInteger) (con integer 10) ] (con integer 7) ]
+    ]
+    (con integer 7)
+  ]
```

Comment:
How was this right before this change? I'm looking at the test description and code in `testCapture` and the previous output doesn't seem right, wouldn't it evaluate to `7+7+7`?